app/client/components/consent/PrivacySettings.tsx

```diff
@@ -285,16 +284,29 @@ export class PrivacySettings extends Component<{}, State> {
   }
 
   private buildState(iabVendorList: ParsedIabVendorList): void {
+    if (iabVendorList && iabVendorList.purposes) {
+      // TODO: Trigger error
+      // tslint:disable-next-line: no-console
+      console.log("ERROR: iabVendorList not present");
```

Comment:
The console log doesn't reflect what's happening here, it'll log "iabVendorList not present" when `iabVendorList` and `iabVendorList.purposes` are present? Is this `if` block necessary?

Reply:
also we should probably be using Sentry for this sort of error logging

Reply:
This was meant to cehck if `iabVendorList` and `iabVendorList.purposes` exist to please the type checker but actually we don't need it because of how we typed the function parameters. Will remove on a separate PR.

Reply:
@twrichards we have a card to integrate Sentry, these logs provide placeholders for now whilst this is in development but they'll all be removed soon.